laa_court_data_api_app/internal/court_data_adaptor_client.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from typing import Optional
+
+import httpx
+from fastapi import Depends
+
+from ..config.court_data_adaptor import CdaSettings
+from ..internal.oauth_client import OauthClient
+
+logger = logging.getLogger(__name__)
+
+
+async def get(endpoint: str, params: Optional[dict[str, str]] = None, headers: Optional[dict[str, str]] = None,
+              oauth_client: OauthClient = Depends(OauthClient), settings: CdaSettings = Depends(CdaSettings)):
```

Comment:
I've noticed a difference in usage of the CdaSettings between here and the oauth client. I think the Depends here needs to call get_cda_settings() similar to line 30 of the oauth client

Reply:
the get_cda_settings() mirrors get_app_settings() of the app.py file in config - this instantiation function might actually be redundant - so maybe we can clean our code up by just using this format throughout?